benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -107,6 +174,14 @@ def run_once(model: Callable, inp: InputsType, task: str, v: VType) -> None:
     else:
         res = func(model, inp, strict=True)
 
+def run_once_functorch(model: Callable, inp: InputsType, task: str, v: VType) -> None:
```

Comment:
@vfdev-5: are the inputs guaranteed to have requires_grad=False?

Functorch transforms differ from torch.autograd.functional in that torch.autograd.function default to create_graph=False but functorch transforms all have an implicit create_graph=True.

If inputs have requires_grad=False then there is no difference, but if inputs have requires_grad=True then this is no longer an apples to apples comparison.

If the inputs have requires_grad=True then we probably want to change them to have requires_grad=False before passing them into a functorch transform OR run the functorch transform underneath with `with torch.no_grad():`

Reply:
Yes, thanks for pointing out this. Indeed, inputs have requires_grad=True. I'll an appropriate context manager for functorch part.